### PR TITLE
Feature: Add pen tool icon

### PR DESCRIPTION
**Pull request by ashygee**
screenshot 2019-01-26 14 47 23

Suggesting to add a pen tool icon to collection.

Request review @colebemis.

### CODE REVIEW

**Comment by colebemis** [comment]
Looks good! I usually try to avoid using the "dot" if I can help it. What do you think of these options? Do you have a preference? 
image

**Comment by ashygee** [comment]
@colebemis i like the first version you made. What would that look like with the nib widened a bit?

**Comment by colebemis** [comment]
Which one do you like?
image

**Comment by ashygee** [comment]
@colebemis I think for Feather the first is the best choice. It keeps the angles and simplicity of the other icons.

**Comment by colebemis** [comment]
Sweet. Let's roll with that one. The icon is in the Figma doc 👍

**Comment by ashygee** [comment]
Updated with the new version. Ready for review @colebemis!

File: src/tags.json [hunk 1]
```diff
@@ -103,6 +103,7 @@
   "paperclip": ["attachment"],
   "pause": ["music", "stop"],
   "pause-circle": ["music", "stop"],
+  "pen-tool": ["pen", "vector", "drawing"],
```
Comment (colebemis) [inline src/tags.json]: "pen" doesn't need to be in the list because it's in the name 👍 

```suggestion
  "pen-tool": ["vector", "drawing"],
```

**Pull request merged**